src/main/java/com/stripe/model/ApplePayDomain.java

```diff
@@ -0,0 +1,110 @@
+import com.stripe.exception.InvalidRequestException;
+import com.stripe.net.APIResource;
+import com.stripe.net.RequestOptions;
+
+import java.util.Map;
+
+public class ApplePayDomain extends APIResource implements HasId {
+	String id;
+	Long created;
+	String domainName;
+	Boolean livemode;
+
+	public String getId() {
+		return id;
+	}
+
+	public void setId(String id) {
+		this.id = id;
+	}
+
+	public Long getCreated() {
+		return created;
+	}
+
+	public void setCreated(Long created) {
+		this.created = created;
+	}
+
+	public Boolean getLivemode() {
+		return livemode;
+	}
+
+	public void setLivemode(Boolean livemode) {
+		this.livemode = livemode;
+	}
+
+	public String getDomainName() {
+		return domainName;
+	}
+
+	public void setDomainName(String domainName) {
+		this.domainName = domainName;
+	}
+
+	public static ApplePayDomain create(Map<String, Object> params)
+			throws AuthenticationException, InvalidRequestException,
+			APIConnectionException, CardException, APIException {
+		return create(params, (RequestOptions) null);
+	}
+
+	public static ApplePayDomain retrieve(String id) throws AuthenticationException,
+			InvalidRequestException, APIConnectionException, CardException,
+			APIException {
+		return retrieve(id, (RequestOptions) null);
+	}
+
+	public DeletedApplePayDomain delete() throws AuthenticationException,
+			InvalidRequestException, APIConnectionException, CardException,
+			APIException {
+		return delete((RequestOptions) null);
+	}
+
+	public static ApplePayDomainCollection list(Map<String, Object> params)
+			throws AuthenticationException, InvalidRequestException,
+			APIConnectionException, CardException, APIException {
+		return list(params, (RequestOptions) null);
+	}
+
+	public static ApplePayDomain create(Map<String, Object> params, RequestOptions options)
+			throws AuthenticationException, InvalidRequestException,
+			APIConnectionException, CardException, APIException {
+		return request(RequestMethod.POST, getClassURL(), params, ApplePayDomain.class, options);
+	}
+
+	public static ApplePayDomain retrieve(String id, RequestOptions options)
+			throws AuthenticationException, InvalidRequestException,
+			APIConnectionException, CardException, APIException {
+		return request(RequestMethod.GET, getInstanceURL(id), null, ApplePayDomain.class, options);
+	}
+
+	public DeletedApplePayDomain delete(RequestOptions options) throws AuthenticationException,
+			InvalidRequestException, APIConnectionException, CardException,
+			APIException {
+		return request(RequestMethod.DELETE, getInstanceURL(id), null, DeletedApplePayDomain.class, options);
+	}
+
+	public static ApplePayDomainCollection list(Map<String, Object> params, RequestOptions options)
+			throws AuthenticationException, InvalidRequestException,
+			APIConnectionException, CardException, APIException {
+		return requestCollection(getClassURL(), params, ApplePayDomainCollection.class, options);
+	}
+
+	public static String getClassURL() {
```

Comment:
I notice that most of the other models with this method don't use `static`. Would you mind making it an instance method for consistency?

Also, can you see if you can change it to private visibility? I don't think there's any reason to leak this interface out of the package.